test/json_test.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var path = require('path');
+var testCase  = require('nodeunit').testCase;
+var json = require('../lib/reporters/json');
+
+exports.reporters = testCase({
+  'json reporter': testCase({
+    'when given empty result': function(test) {
+      var result = [],
+        reporter = json.reporter,
+        expected = '[]',
+        actual = reporter(result);
+      test.equal(actual, expected, 'Should return empty json array for empty result');
+      test.done();
+    },
+    'when given non-empty result': function(test) {
+      var invalid_html = path.join('test', 'invalid.html'),
+        result = [{
```

Comment:
This test data is duplicated across multiple tests - maybe extract that into another `test/support/...` file?